content/en/docs/contributing/e2e.md

```diff
@@ -5,91 +5,97 @@ weight: 70
 type: "docs"
 ---
 
-cert-manager has an end-to-end test suite that verifies functionality against a
+cert-manager has an extensive end-to-end (e2e) test suite that verifies functionality against a
 real Kubernetes cluster.
 
-This test takes around 30 minutes, it will be run on every PR in our cluster.
+The full end-to-end test suite can take a long time to complete and is run against every pull
+request made to the cert-manager project.
 
-> Note: you can see the status of each commit on the master branch at
-> [`testgrid.k8s.io`](https://testgrid.k8s.io/jetstack-cert-manager-master).
-> You can join the Google group
-> [`cert-manager-dev-alerts`](https://groups.google.com/g/cert-manager-dev-alerts)
-> in order to receive a notification by email whenever a commit on master
-> fails.
+Unless you've made huge changes to the cert-manager codebase - or to the end-to-end
+tests themselves - you probably don't _need_ to run the tests locally. If you do want to
+run the tests, though, this document explains how.
 
-It is only advised to run this locally when you made big changes to the
-codebase. This document explains how you can run the end-to-end tests yourself.
+> Note: the status of each commit on the master branch is reported on
+> [`testgrid.k8s.io`](https://testgrid.k8s.io/jetstack-cert-manager-master).
+> Join the [`cert-manager-dev-alerts`](https://groups.google.com/g/cert-manager-dev-alerts)
+> Google group to receive email notifications when tests fail.
```

Comment:
nit: would it be worth using a shortcode here? (not sure if my proposition is an improvement or not aesthetically wise...)

```suggestion

{{% alert title="TestGrid alerts" color="" %}}

The status of each commit on the master branch is reported on
[`testgrid.k8s.io`](https://testgrid.k8s.io/jetstack-cert-manager-master). Join
the
[`cert-manager-dev-alerts`](https://groups.google.com/g/cert-manager-dev-alerts)
Google group to receive email notifications when tests fail.

{{% /alert %}}
```

Before:

![image](https://user-images.githubusercontent.com/2195781/126308058-86070199-c07c-4d2a-be55-5b30fed133a1.png)

After:

![image](https://user-images.githubusercontent.com/2195781/126308000-babe3c75-c45c-4669-9add-8a697db25063.png)

Reply:
oh yeah, that's much better. I'll use that, thanks :+1: